### DISCUSSION TITLE
Do I need to reinstall if I resize the disk?

### DISCUSSION THREAD

**Discussion by suzushi-tw** [Dokploy/dokploy#563] [Q&A] [answered]
Hi, I run out of space so I resized the disk, but I am not sure how to let dokploy 
detect the new space. Or is it only possible through a reinstall? 
Thanks in advance.

**Comment by Siumauricio** [comment, marked as answer]
Dokploy will not really know if you add more or less space, I think that depends on docker, the library that we use internally is responsible for reading all the available space in the path /

https://github.com/Dokploy/dokploy/blob/canary/packages/server/src/monitoring/utilts.ts#L41

**Reply by suzushi-tw** [reply]
thanks a lot, got it working by merging the disk
